bifrost/release.go

```diff
@@ -29,9 +34,10 @@ type Release struct {
 	UUID      *string `json:"uuid,omitempty"`       // Generated By server
 	ReleaseID *string `json:"release_id,omitempty"` // Generated Client
 
-	ProjectName *string `json:"project_name,omitempty"`
-	ConfigName  *string `json:"config_name,omitempty"`
-	Bucket      *string `json:"bucket,omitempty"` // Bucket with Additional Data in it
+	ProjectName   *string `json:"project_name,omitempty"`
+	ConfigName    *string `json:"config_name,omitempty"`
+	Bucket        *string `json:"bucket,omitempty"`          // Bucket with Additional Data in it
+	LockTableName *string `json:"lock_table_name,omitempty"` // DynamoDB table used for grabbing, and releasing locks
```

Comment:
The lock table name should be calculated from the name of the lambda. Changing this struct will break many of our deployers.

Reply:
Which field does that correspond to? Because it seems to be available in `deployer.Release` but not `bifrost.Release`.